ta/os_test_lib_dl/os_test_lib_dl.c

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2018, Linaro Limited
+ */
+
+#include "os_test_lib_dl.h"
```

Comment:
would prefer `<>` rather than `""` here (file's not in the same directory)

Reply:
yup